IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -503,13 +503,14 @@ - (BOOL)checkAccountIdentifier:(NSString *)accountIdentifier
     return YES;
 }
 
-- (void)fillInternalErrorWithMessage:(NSString *)message
+- (BOOL)fillInternalErrorWithMessage:(NSString *)message
```

Comment:
Why it needs to be BOOL? Seems we always return YES from this method...

Reply:
Fixing the warning... a method that accepts NSError should return BOOL. Seemed weird to me too, but I figured if we added more logic to this method in future, it will make more sense. I'm fine both ways: leaving the warning or returning YES for now.

Reply:
I see, lets keep it bool then